roles/custom/matrix-jitsi/defaults/main.yml

```diff
@@ -205,6 +205,9 @@ matrix_jitsi_jicofo_component_secret: ''
 matrix_jitsi_jicofo_auth_user: focus
 matrix_jitsi_jicofo_auth_password: ''
 
+# Enable Sentry DSN
+matrix_jitsi_jicofo_sentry_dns: ''
```

Comment:
Is this meant to be set by people to some connection string? At least that's my expectation when I see "DSN".

So, to enable the Sentry feature, one needs to set this to a non-empty string, right?

-----

Later on, we never seem to pass the connection string to Jitsi.. We only seem to pass `0` or `1` depending on this variable. How will Jitsi pick up the actual DSN connection string value then?

----

Your PR fixes the defaults, so that we'll be passing `SENTRY_DSN=0` to disable the Sentry feature (by default).

If people want to use Sentry though, they won't be able to do it, would they? I haven't confirmed this, but my understanding is that Jitsi expects either a string value (a valid Sentry DSN connection string) or `0` (to disable Sentry integration).

If that's the case, then maybe it's better to:

- set `matrix_jitsi_jicofo_sentry_dns` and `matrix_jitsi_jvb_sentry_dns` to `0` by default
- adjust the `env.j2` files to pass the `_sentry_dsn` value as is (no `if` stuff necessary)
- adjust the comments for these variables to say that people who wish to enable Sentry should override this and specify a valid Sentry DSN connection string value instead of this `0`

By the way, there's also a typo in the variable names (`_sentry_dns` -> `_sentry_dsn`).